types/angular-sanitize/index.d.ts

```diff
@@ -12,39 +13,85 @@ import * as angular from 'angular';
 declare module 'angular' {
     ///////////////////////////////////////////////////////////////////////////////
     // ngSanitize module (angular-sanitize.js)
+    // see https://code.angularjs.org/1.7.0/docs/api/ngSanitize
     ///////////////////////////////////////////////////////////////////////////////
     namespace sanitize {
-        ///////////////////////////////////////////////////////////////////////////
-        // SanitizeService
-        // see http://docs.angularjs.org/api/ngSanitize.$sanitize
-        ///////////////////////////////////////////////////////////////////////////
+        /**
+         * Sanitizes an html string by stripping all potentially dangerous tokens.
+         *
+         * The input is sanitized by parsing the HTML into tokens.
+         * All safe tokens (from a whitelist) are then serialized back to a properly escaped HTML string.
+         * This means that no unsafe input can make it into the returned string.
+         *
+         * The whitelist for URL sanitization of attribute values is configured using the functions aHrefSanitizationWhitelist and imgSrcSanitizationWhitelist of $compileProvider.
+         * The input may also contain SVG markup if this is enabled via $sanitizeProvider.
+         *
+         * @param html HTML input.
+         */
         interface ISanitizeService {
             (html: string): string;
         }
 
+        /**
+         * Creates and configures $sanitize instance.
+         */
+        interface ISanitizeProvider {
+            /**
+             * Enables a subset of svg to be supported by the sanitizer.
+             *
+             * @see https://code.angularjs.org/1.7.0/docs/api/ngSanitize/provider/$sanitizeProvider#enableSvg
+             * @param flag Enable or disable SVG support in the sanitizer.
+             */
+            enableSvg(flag: boolean): boolean;
```

Comment:
I think the following set of overloads is more appropriate:
```ts
enableSvg(): boolean;
enableSvg(flag: boolean): ISanitizeProvider;
enableSvg(flag?: boolean): boolean | ISanitizeProvider;
```

Reply:
Hi @uniqueiniquity!
From docs:

> Returns the currently configured value if called without an argument or self for chaining otherwise.

Why do you think your set of of overloads is more appropriate? Because this `enableSvg(flag: boolean): ISanitizeProvider;` is not true.

Did You meant this?
```
enableSvg(): ISanitizeProvider;
enableSvg(flag: boolean): boolean;
enableSvg(flag?: boolean): boolean | ISanitizeProvider;
```